wordpress-mobile/WordPress-Android
# Setting FAB visibility in onResume.

Fixes #12830 

This PR moves the FAB visibility setting to onResume of the main activity. This lets us react to changes in the currently selected site and update the visibility of the FAB accordingly.

To test:
- Sign up with a new account, and proceed to the "Welcome to WordPress" screen.
- Tap the "Create WordPress.com site" button and create a site.
- After you are brought to My Site screen, notice that the FAB is visible.
- Navigate to Site Settings and delete your site.
- After you are brought to My Site screen, notice that the FAB is not visible.
- From empty My Site screen create a new site.
- After you are brought back to My Site screen, notice that the FAB is visible.


PR submission checklist:
